HacktoberfestProject.Web/styles/raw.css

```diff
@@ -3,3 +3,50 @@
 @tailwind components;
 
 @tailwind utilities;
+
+/*Twitter Typeahead
```

Comment:
Temporary inclusion of styles for typeahead here so they are more than just text that overlaps with UI elements under them.  As @whitep4nth3r implements the new styling, she can decide if this needs replaced or moved, etc.